Dont fatal if saving state failed.
this alway fails on qemu.

diff -Nur MPlayer-20060519.orig/libvo/vo_vesa.c MPlayer-20060519/libvo/vo_vesa.c
--- MPlayer-20060519.orig/libvo/vo_vesa.c	Sat Apr 29 02:06:42 2006
+++ MPlayer-20060519/libvo/vo_vesa.c	Fri May 26 18:30:30 2006
@@ -923,7 +923,6 @@
 		if((err=vbeSaveState(&init_state)) != VBE_OK)
 		{
 			PRINT_VBE_ERR("vbeSaveState",err);
-			return -1;
 		}
 		
 		/* TODO: 
